DMCompiler/Compiler/DM/DMParser.cs

```diff
@@ -1072,7 +1198,7 @@ public DMASTStatement Statement(bool requireDelimiter = true)
                     DMASTProcStatement statement = ProcStatement();
                     if (statement == null) Error("Expected statement");
 
-                    body = new DMASTProcBlockInner(loc, new DMASTProcStatement[] { statement });
+                    body = new DMASTProcBlockInner(loc, new DMASTProcStatement[] { statement }, new DMASTProcStatement[] { });
```

Comment:
The constructor will replace `null` with `Array.Empty<DMASTProcStatement>()`
```suggestion
                    body = new DMASTProcBlockInner(loc, new DMASTProcStatement[] { statement }, null);
```